mosaic2/grid2.F90

```diff
@@ -94,7 +94,8 @@ module grid2_mod
      MAX_FILE = 1024, & ! max length of the file names
      VERSION_0 = 0,   &
      VERSION_1 = 1,   &
-     VERSION_2 = 2
+     VERSION_2 = 2,   &
+     VERSION_3 = 3
```

Comment:
Do you know what these are?  It would be great to have them documented.

Reply:
I don't... I can glean certain information about them (e.g. versions 0 and 1 seem to be "pre cubed sphere"), but the only test used to differentiate them is the existence of certain variables within the file.

Reply:
OK, I will approve, but this needs to be documented